scenes/game_elements/props/buildings/house/house_1.tscn

```diff
@@ -9,6 +9,9 @@ size = Vector2(64.0002, 80)
 [node name="House_1" type="Node2D"]
 script = ExtResource("1_qufpk")
 house_sprite__texture = ExtResource("2_qufpk")
+house_sprite__flip_h = false
+house_sprite__flip_v = false
+house_sprite__offset = Vector2(0, 0)
```

Comment:
I think this diff in the decoration scenes happen because this is being exposed through the PropertyUtils, not through regular exporting. If the properties are the default ones, Godot doesn't save them in the scene.

Reply:
Hmm... I think this can be fixed by using the [`_property_can_revert`](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-private-method-property-can-revert)and the [`_property_get_revert`](https://docs.godotengine.org/en/stable/classes/class_object.html#class-object-private-method-property-get-revert) methods.

We can add an issue if it's important to solve it!